polkadot-parachains/src/chain_spec.rs

```diff
@@ -211,19 +177,19 @@ fn integritee_genesis_config(
 	parachain_id: ParaId,
 ) -> serde_json::Value {
 	serde_json::json!({
-		"collatorSelection": integritee_runtime::CollatorSelectionConfig {
+		"collatorSelection": integritee_kusama_runtime::CollatorSelectionConfig {
```

Comment:
all hail the untyped json 🙈

Reply:
yeah, well. also this will be replaced similar to the fellowship runtimes repo.... but for now, I guess, this is good enough